src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -33,11 +33,12 @@ internal sealed class MsgPackLogExporter : MsgPackExporter, IDisposable
     private readonly IReadOnlyDictionary<string, object> m_customFields;
 
     private readonly ExceptionStackExportMode m_exportExceptionStack;
+    private readonly ExporterStateForScopes exporterStateForScopes;
```

Comment:
nit: `m_exporterStateForScopes` to be consistent with the other fields.

Reply:
Updated. I think we should update the field names to use Pascal case instead though. 😄